crates/repo/src/lib.rs

```diff
@@ -210,3 +210,42 @@ pub fn app() -> Router {
         }),
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use namespace::Namespace;
+    use std::net::{IpAddr, Ipv4Addr, SocketAddr, TcpListener};
+
+    #[tokio::test]
+    async fn routes() {
+        tokio::spawn(async move {
```

Comment:
@rvolosatovs The client fails to connect and I think there is an issue here getting the connection opened.